actors/builtin/market/market_state.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/filecoin-project/specs-actors/actors/util/adt"
 )
 
-const epochUndefined = abi.ChainEpoch(-1)
+const EpochUndefined = abi.ChainEpoch(-1)
```

Comment:
I'd define this in `abi/primitives.go`, if we're going to export it.

Reply:
Yeah, minor preference for not exporting it, but if we do, move it to `abi.`

Reply:
I don't think we need to export this. Restoring this.